src/main/java/uk/ac/ebi/ampt2d/metadata/rest/controllers/StudyRestController.java

```diff
@@ -157,10 +156,10 @@
     }
 
     @ApiOperation(value = "Get a list of studies linked to a given study")
-    @ApiParam(name = "id", value = "Study's id (accession.version)", type = "string", required = true, example = "EGAS0001.1")
+    @ApiParam(name = "id", value = "StudyId", type = "string", required = true)
     @RequestMapping(method = RequestMethod.GET, path = "{id}/linkedStudies", produces = "application/json")
     @ResponseBody
-    public ResponseEntity<Resources<StudyResource>> getLinkedStudies(@PathVariable("id") AccessionVersionEntityId id) {
```

Comment:
Please do not remove support for queries by accession+version, which is the public identifier for an object.

Reply:
after discussing with Jorge , any object can be created without an accession , so i think accession cannot be used as an identifier , we can have a queryByAccession but not as an identifier . And also i need to define uniqueConstraints for the objects to stop updates using Post. So as accessions can be null i can't use it as a unique constraint now .

Reply:
Yeah well, call it "public alias" instead of "public identifier" if you prefer to do so.

> So as accessions can be null i can't use it as a unique constraint now .

Read the last paragraph from [this section](https://www.postgresql.org/docs/current/static/ddl-constraints.html#DDL-CONSTRAINTS-UNIQUE-CONSTRAINTS). The SQL standard supports unique constraints in nullable columns.

Reply:
Yes but then , the goal of the jira stop update using post method can't be achieved i guess , same post will create new object (not even merge).